components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -111,15 +113,29 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _displayMaskLayer.fillColor = [UIColor whiteColor].CGColor;
 
     _titleLabel = [[UILabel alloc] initWithFrame:CGRectZero];
-    _titleLabel.font = [UIFont mdc_standardFontForMaterialTextStyle:MDCFontTextStyleTitle];
+    // If we are using the default (system) font loader, retrieve the
+    // font from the UIFont standardFont API.
```

Comment:
We need additional info:
Since mdc_standardFont... returns a different value than [MDCTypography titleFont] we are jumping through some extra logic to use the newer font sizes in mdc_standardFont... Once [MDCTypograhy titleFont] returns the same value as mdc_standardFont... we can remove some of this logic.

Reply:
Added a link to an issue.  Trying to embed the explanation in a comment seems a bit too much.

Reply:
sgtm